app/components/LargeHeader.js

```diff
@@ -19,7 +26,7 @@ function LargeHeader() {
         <div className="search-container form-row" role="search">
           <input type="text" className="search-field" placeholder="I'm looking for..." name="srch-term" id="srch-term" />
           <input type="text" className="location-field" placeholder="Near me" name="srch-location" id="srch-location" />
-          <button id="largeheader_searchbutton" className="button search" type="submit"><i className="material-icons">search</i> <span>Search</span></button>
+          <button onClick={submitSearch} id="largeheader_searchbutton" className="button search" type="submit"><i className="material-icons">search</i> <span>Search</span></button>
```

Comment:
We're in a form, yeah? Is it `onSubmit` On the form tag? Then the button should have a "submit" type attribute. Not 100%, that's just what a quick Google search turned up.